src/index.css

```diff
@@ -0,0 +1,593 @@
+  display: grid;
+  grid-template-columns: 4fr 1fr;
+  grid-gap: 10px;
+  margin: 3em 5em;
+  padding: 2em;
+  border: 2px solid var(--colour-biscuit);
+}
+
+.special-menu-header {
+  grid-column: 1 / -1;
+  color: var(--colour-sage);
+  padding: 0%;
+  margin: 0%;
+  font-size: 35px;
+}
+
+.special-menu {
+  grid-column: 1 / -1;
+  color: var(--colour-sage);
+  font-size: 20px;
+}
+
+/* reserve a table */
+.menu-reservation {
+  background-image: url("./images/Menu/Background.png");
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: var(--colour-white);
+  min-height: 500px;
+  background-size: cover;
+}
+
+/* ---------------------- */
+/* Reservation page starts here  */
+.reservation-hero {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url("./images/Reservations/Hero.png");
+  font-family: "Cormorant", serif;
+  min-height: 500px;
+  background-size: cover;
+}
+
+.logo-img {
+  margin: 20px 0 0 20px;
+}
+
+.form-title {
+  margin: 20px 0 0 20px;
+}
+
+/* reservation form starts here  */
+.form-container {
+  /* width: 100%;
+  height: auto; */
+  display: grid;
+  grid-template-columns: repeat(2,100px);
+  grid-auto-rows: repeat(10, 50px);
+  grid-template-areas:
+    "l1 l2"
+    "i1 i2"
+    "l3 ."
+    "i3 ."
+    "l4 l4"
+    "i4 i4"
+    "l5 l5"
+    "i5 i5"
+    "l6 l6"
+    "ta ta";
+  grid-gap: 10px;
+  margin: 2em;
+  min-width: 370px;
+}
+
+.label1 {
+  grid-area: l1;
+}
+
+.input1 {
+  grid-area: i1;
+}
+
+.label2 {
+  grid-area: l2;
+}
+
+.input2 {
+  grid-area: i2;
+}
+
+.label3 {
+  grid-area: l3;
+}
+
+.input3 {
+  grid-area: i3;
+}
+
+.label4 {
+  grid-area: l4;
+}
+
+.input4 {
+  grid-area: i4;
+}
+
+.label5 {
+  grid-area: l5;
+}
+
+.input5 {
+  grid-area: i5;
+}
+
+.label6 {
+  grid-area: l6;
+}
+
+.textarea {
+  grid-area: ta;
+  padding: 50px;
+  border: 1px solid var(--colour-sage);
+  border-radius: 5px;
+}
+
+.checkbox {
+  display: flex;
+  flex-direction: column;
+}
+
+.box {
+  margin-left: 30px;
+}
+
+.button-submit {
+  text-align: center;
+}
+
+.btn-submit {
+  background-color: var(--colour-sage);
+  color: var(--colour-white);
+  margin: 20px 0px 40px 0px;
+  padding: 10px 30px;
+  border: 0ch;
+}
+
+label {
+  color: var(--colour-sage);
+}
+
+input {
+  border: 1px solid var(--colour-sage);
+  border-radius: 5px;
+  padding: 10px;
+}
+
+/* .main-div {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+} */
+
+/* reservation menu here  */
+.menu-items {
+  background-image: url("./images/Reservations/Background.png");
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: var(--colour-white);
+  min-height: 500px;
+  background-size: cover;
+}
+
+.menu-btn {
+  color: var(--colour-sage);
+  padding: 15px;
+}
+
+.img2-div {
+  display: none;
+}
+
+/* ----------Media query section starts here-------- */
+/* ----------- media query for tablet------*/
+@media only screen and (min-width: 400px) {
```

Comment:
This is a little early to start tablet. Small tablets start around 600px wide, but many codebases start tablet designs from 768px wide (the width of early ipad versions)